react/features/shared-video/components/web/SharedVideoDialog.js

```diff
@@ -110,6 +111,14 @@ class SharedVideoDialog extends AbstractSharedVideoDialog<*> {
         }
 
         const youtubeId = getYoutubeId(link);
+
+        if (!youtubeId) {
```

Comment:
this will prevent users from sharing anything other than youtube videos

Reply:
Exactly. This was the requirement.

Reply:
@quitrk Do we let users share other videos ? I wasn't sure if there is such a feature. We got a report that the player shows up even when a random link is shared and that the client doesn't throw an error. If you are aware of another valid use case please let us know.

Reply:
https://github.com/jitsi/jitsi-meet/commit/4e4ff0f60f7b823864722567fb882b15f99b631f

We allow user share direct video links like mp4  for example

Reply:
You could use these:
https://github.com/jitsi/jitsi-meet/blob/master/react/features/shared-video/components/web/YoutubeVideoManager.js#L213
https://github.com/jitsi/jitsi-meet/blob/master/react/features/shared-video/components/web/VideoManager.js#L165

And implement the. error handling in [AbstractVideoManager](https://github.com/jitsi/jitsi-meet/blob/master/react/features/shared-video/components/web/AbstractVideoManager.js) which is extended by those two

Reply:
ah ok, thanks for letting us know. @tudor-phd, can you please implement error handling in the AbstractVideoManager like @quitrk suggested ?

Reply:
Sure. Added some changes